libs/api/Verifier.cpp

```diff
@@ -29,8 +29,18 @@ typedef struct _section_program_map
     string program_name;
 } section_program_map_t;
 
+typedef struct _section_offset_to_map
+{
+    size_t section_offset;
+    string map_name;
+} section_offset_to_map_t;
+
 static void
-_get_section_and_program_name(string& path, vector<section_program_map_t>& map) noexcept(false)
+_get_program_and_map_names(
+    _In_ string& path,
+    _Inout_ vector<section_program_map_t>& section_to_program_map,
+    _Inout_ vector<section_offset_to_map_t>& map_names,
```

Comment:
Ok for now but should really use a map not a vector.  Ok to add it to issue #211